### PR TITLE
Add more switcher script aliases

### PR DESCRIPTION
**Pull request by 3noch**
An attempt to really nail down #27.

Again, open to improvements.

### CODE REVIEW

File: Installer.hs [hunk 1]
```diff
@@ -72,7 +70,25 @@ installer arch quick version = nsis $ do
...
+              ]
 
 showArchAbbr :: Arch -> String
 showArchAbbr Arch32 = "32"
 showArchAbbr Arch64 = "64"
+
+
+majorVersion :: Version -> Version
+majorVersion ver =
+    if length parts >= 3
```
Comment (ndmitchell) [inline Installer.hs]: Is the condition necessary? In the case parts is short, take 2 will have no effect.

File: Installer.hs [hunk 2]
```diff
@@ -72,7 +70,25 @@ installer arch quick version = nsis $ do
         -- make sure we don't remove $APPDATA/cabal/bin, since users may have had that on their $PATH before
         mapM_ (setEnvVarRemove HKCU "PATH") $ "$INSTDIR/switch" : tail path
 
+    where
+        switcherNameSuffixes
+            = "" -- no suffix
+            : ['-' : intercalate "-" parts
```
Comment (ndmitchell) [inline Installer.hs]: Could be concatMap ('-':) parts, which is a fraction simpler.

**Comment by ndmitchell** [comment]
Looks very good. A few minor Hlint style changes, but happy to merge as is I you'd prefer.

File: Installer.hs (hunk 1 above)
Comment (3noch) [inline Installer.hs]: You're right. I didn't realize that. `head` has always left me in a state of "partial paranoia" (pun intended).

File: Installer.hs (hunk 2 above)
Comment (3noch) [inline Installer.hs]: Ah yes, much simpler. Thanks.

**Comment by 3noch** [comment]
@ndmitchell Rebased in some changes. Happy to make further improvements as you see fit.

**Comment by ndmitchell** [comment]
Looks good. One thought, is the Arch abbrev constant now redundant? (I'm on holiday with a mobile so can't see if it's still there)

**Comment by 3noch** [comment]
Indeed it is. Good catch.

**Comment by 3noch** [comment]
Fixed. Good Good Friday to you. Indeed this is a holy-day. :)

**Comment by ndmitchell** [comment]
Where I am is the Russian orthodox so Easter is a week later :). Thanks for all te changes, they look great.

**Comment by ndmitchell** [comment]
Sorry, I had a memory of clicking merge on my mobile, so either it was my phone or my brain had an off day. Now merged. Thanks!

**Pull request merged**